examples/mnist/pipeline/mnist_spark.py

```diff
@@ -99,18 +96,36 @@ def toNumpy(bytestr):
 
 model = estimator.fit(df)
 
+#
+# Using signature defs
+#
+
 # prediction
-model.setTagSet(tf.saved_model.tag_constants.SERVING) \
-      .setSignatureDefKey(tf.saved_model.signature_constants.DEFAULT_SERVING_SIGNATURE_DEF_KEY) \
-      .setInputMapping(['col1=image']) \
-      .setOutputMapping(['col_out=prediction']) \
+# model.setTagSet(tf.saved_model.tag_constants.SERVING) \
+#       .setSignatureDefKey(tf.saved_model.signature_constants.DEFAULT_SERVING_SIGNATURE_DEF_KEY) \
+#       .setInputMapping(['col1=image']) \
+#       .setOutputMapping(['col_out=prediction']) \
 
 # featurize
-#model.setTagSet(tf.saved_model.tag_constants.SERVING) \
+# model.setTagSet(tf.saved_model.tag_constants.SERVING) \
 #      .setSignatureDefKey('featurize') \
 #      .setInputMapping(['col1=image', 'col2=label']) \
 #      .setOutputMapping(['col_out=features'])
 
+#
+# Using custom/direct mappings
+#
+
+# prediction
+# model.setTagSet(tf.saved_model.tag_constants.SERVING) \
+#       .setInputMapping(['col1=x']) \
+#       .setOutputMapping(['col_out=prediction'])
+
+# featurize
+model.setTagSet(tf.saved_model.tag_constants.SERVING) \
+      .setInputMapping(['col1=x', 'col2=y_']) \
+      .setOutputMapping(['col_out=Relu'])
+
```

Comment:
@leewyang Can you add an example that construct TFModel directly from an export or checkpoint file?